### PR TITLE
bug 2093957: [docs] Light readme maint

### PR DESCRIPTION
**Pull request by afflom**
# Description

Inform users of pruning behavior and mode compat. also change meet freq

Fixes # (issue)

## Type of change

Docs/Readme

### CODE REVIEW

**Comment by jpower432** [comment]
@afflom It may be worth it to add information about environment workflow changes. This BZ https://bugzilla.redhat.com/show_bug.cgi?id=2093957 could be addressed with some additional information about how metadata is used and why oc-mirror stores it in specific places to allow the changes from fully disconnected to partially disconnected and vice-versa.

**Comment by jpower432** [comment]
@afflom It may be worth it to add information about environment workflow changes. This BZ https://bugzilla.redhat.com/show_bug.cgi?id=2093957 could be addressed with some additional information about how metadata is used and why oc-mirror stores it in specific places to allow the changes from fully disconnected to partially disconnected and vice-versa.

File: README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@
 `oc-mirror` is an OpenShift Client (oc) plugin that manages OpenShift release, operator catalog, helm charts, and associated container images for mirror registries that support OpenShift environments.
 
 ## Community
-Please join us for our oc-mirror weekly community meeting Thursdays at 12pm EST https://zoom.us/j/99871298397?pwd=eHYrYlQxTDJaOTQ2bDl3VXlRMjFpUT09 . Retrieve the meeting passcode from the first comment in the following issue: https://github.com/openshift/oc-mirror/issues/349 . 
+Please join us for our oc-mirror weekly community meeting fortnightly on Thursdays at 12pm EST https://zoom.us/meeting/tJ0lcOiqpzMpE9OQ8gwJktuKL7ZWAr3Ofe4s/ics?icsToken=98tyKuChpjwrGN2dsxCARox5Go_Cd-rxmFxej_pyrUfiFApYdhbdGrV4BZNtQ_fW .
```
Comment (jpower432) [inline README.md:22]: It may be more helpful if there was a calendar linked since this is not weekly.

**Review by jpower432 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (afflom) [reply README.md:22]: That is a calendar link

**Pull request merged**